### DISCUSSION TITLE
Unable to access IPFS api

### DISCUSSION THREAD

**Discussion by DevAshish-oodles** [filecoin-project/community#283] [Developers]
I am getting 404 error when trying to reach IPFS api or trying to connect with IPFS port 9090

**Comment by mishmosh** [comment]
@DevAshish-oodles Can you share the code you're using to try to connect?

**Comment by DevAshish-oodles** [comment]
I have configured the server from default documentation provided for
go-ipfs installation for CLI on official website, but i am facing issue to
access localhost:5001 to use GET API as it is showing CORS error. ALthough
i have enabled (ipfs config --json
API.HTTPHeaders.Access-Control-Allow-Origin '["*"]) it for all.

On Thu, Sep 9, 2021 at 6:57 PM Mosh ***@***.***> wrote:

> @DevAshish-oodles <https://github.com/DevAshish-oodles> Can you share the
> code you're using to try to connect?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/filecoin-project/community/discussions/283#discussioncomment-1301576>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUA3TUIHPM3CUHWEOW3NGW3UBCY5RANCNFSM5DI6CEGQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
<https://www.oodlestechnologies.com/>
*Ashish Sharma*Sr. Associate Consultant - DevOps
+91  9691901988ashish.sharma

technologies.oodles.io | www.oodlesmarketing.com |
www.oodlesstudio.comDG-18-009,
Tower-B, Emaar Digital Greens, Sector- 61, Gurugram, Haryana- 122011, IN.
[image: Facebook icon]
<https://www.linkedin.com/company/oodles-technologies-pvt-ltd> [image:
Youtbue icon] <https://www.facebook.com/OodlesTech> [image: LinkedIn icon]
<https://twitter.com/oodlestech>

This e-mail message may contain confidential or legally privileged
information and is intended only for the use of the intended recipient(s).
Any unauthorized disclosure, dissemination, distribution, copying or the
taking of any action in reliance on the information herein is prohibited.
E-mails are not secure and cannot be guaranteed to be error free as they
can be intercepted, amended, or contain viruses. Anyone who communicates
with us by e-mail is deemed to have accepted these risks. Oodles
Technologies is not responsible for errors or omissions in this message and
denies any responsibility for any damage arising from the use of e-mail.
Any opinion and other statement contained in this message and any
attachment are solely those of the author and do not necessarily represent
those of the company
